mars/deploy/local/session.py

```diff
@@ -81,6 +81,11 @@ def run(self, *tensors, **kw):
 
         return data_list
 
+    def fetch(self, key):
+        graph_key = self._tensor_to_graph[key]
+        resp = self._api.fetch_data(self._session_id, graph_key, key)
+        return dataserializer.loads(resp)
```

Comment:
I think the MarsAPI class should handle data serialization / deserialization itself.